src/components/Icon/img/ukraine.svg

```diff
@@ -0,0 +1,38 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" id="ukraine">

```

Comment:
Maybe you could use https://jakearchibald.github.io/svgomg/ to compress this SVG

Reply:
fixed